ethexe/processor/src/handling/run.rs

```diff
@@ -34,31 +34,32 @@ enum Task {
 }
 
 pub fn run(
-    threads_amount: usize,
+    num_workers: usize,
```

Comment:
I do not recommend to do this renaming. Because this is real threads amount in tokio runtime and we have also workers amount which is amount of virtual threads. Of course we can set real threads amount to the amount of workers but better to specify that it is different things.